templates/docs/meter/shelly-1pm_0.yaml

```diff
@@ -11,4 +11,4 @@ render:
       host: 192.0.2.2 # IP-Adresse oder Hostname
       user: # Benutzerkonto (bspw. E-Mail Adresse, User Id, etc.) # Optional
       password: # Passwort des Benutzerkontos (bei führenden Nullen bitte in einfache Hochkommata setzen) # Optional
```

Comment:
In den Templates müssen wir den Hinweis noch von einfache auf doppelte Hochkommata ändern:
`... bei führenden Nullen bitte in doppelte Hochkommata setzen ...`